src/mainboard/amd/romecrb/src/main.rs

```diff
@@ -214,9 +214,9 @@ pub extern "C" fn _start(fdt_address: usize) -> ! {
     uart0.pwrite(b"Welcome to oreboot\r\n", 0).unwrap();
     debug.init().unwrap();
     debug.pwrite(b"Welcome to oreboot - debug port 80\r\n", 0).unwrap();
```

Comment:
To not confuse, when switching to COM2 here, the debug print should probably also say COM2

Reply:
Oh, I was of the impression that the idea was to make the welcome message say which port the respective output is on (so you can find out what is connected to what on the outside of the chip).

I can change it, no problem.

Reply:
Oh right, I confused things here - I'm sorry! So we have `uart0` now, `debug` (which is port 0x80), and `p0` (which is now com2). I think the variables names could be adjusted a bit, but that's probably not for now. :) Just like this it's fine now, imo.